GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -45,6 +44,9 @@ public sealed partial class FormCommit : GitModuleForm
 
         #region Translation
 
+        private readonly TranslationString _commitAndPush = new("Commit && &push");
+        private readonly TranslationString _push = new("&Push");
```

Comment:
Can we reuse existing translations? I.e. move the existing translations into `TranslatedStrings` and reuse them.

Reply:
That is hard with '&' accelerators...

Reply:
There is a match:
https://github.com/gitextensions/gitextensions/blob/63ba58e3579381edc1d374774ba1233bdce4d9d9/GitUI/CommandsDialogs/FormPush.Designer.cs#L146

Reply:
That's cool, wanted to check we didn't duplicate.